### PR TITLE
Bundle proto and openapi

### PR DESCRIPTION
**Pull request by czerwiukk**
## Description

Marks `fishjam-openapi` and `fishjam-proto` packages as private and configures `js-server-sdk` to bundle them.

## Motivation and Context

Publishing the packages mentioned above shouldn't be necessary.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)

### CODE REVIEW

File: packages/fishjam-openapi/package.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "@fishjam-cloud/fishjam-openapi",
   "version": "0.3.0",
+  "private": true,
```
Comment (mironiasty) [inline packages/fishjam-openapi/package.json:4]: you should also update `release.yml` to stop publishing these packages (and probably mark them as deprecated?)

File: packages/fishjam-openapi/src/generated/api.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@
  * Fishjam Media Server
  * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
  *
- * The version of the OpenAPI document: 0.9.0
+ * The version of the OpenAPI document: 0.9.0-dev
```
Comment (mironiasty) [inline packages/fishjam-openapi/src/generated/api.ts:7]: Can we stick to generating api from released versions?

**Review by mironiasty — COMMENTED** [review]
How did you verified that fishjam-openapi is now bundled in `js-server-sdk`? I've run `yarn pack` but I do not see it inside packages 🤔

**Comment by czerwiukk** [comment]
> How did you verified that fishjam-openapi is now bundled in `js-server-sdk`? I've run `yarn pack` but I do not see it inside packages 🤔

The `chunk` and `index` files contain protobuf packing code and openapi axios methods repectively.
I didn't see value in putting effort to configure the packages to bundle to their corresponding files.

**Review by Karolk99 — APPROVED** [review]

**Review by mironiasty — APPROVED** [review]
Ok, in that case I assume you are right and this package will work fine after releasing :)
🚢 🇮🇹

**Pull request merged**